src/home/pam_systemd_home.c

```diff
@@ -698,7 +700,7 @@ _public_ PAM_EXTERN int pam_sm_authenticate(
         if (debug)
                 pam_syslog(handle, LOG_DEBUG, "pam-systemd-homed authenticating");
 
-        return acquire_home(handle, /* please_authenticate= */ true, suspend_please, debug);
+        return acquire_home(handle, /* please_authenticate= */ true, suspend_please, debug, NULL);
```

Comment:
the children are only forked off once the session hook is called. hence all we need to make sure is that once the session hook ends (i.e. pam_sm_open_session()) under all circumstances guarantees that the bus connection is terminated. 

Authentication (i.e. this pam_sm_authenticate() hook) happens much earlier than that though, and we expressly want to share the bus connection between pam_sm_authenticate() and pam_sm_open_session() to avoid unnecessary roundtrips, hence we expressly *don't* want to destory the conneciton here, so that we can still make use of it in pam_sm_open_session().

Reply:
ok